<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(238,238,238)</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="inputSelectButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(52,73,85);
color:rgb(255,255,255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Seleccionar Entrada</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filterSelectButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(52,73,85);
color:rgb(255,255,255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Seleccionar Filtro</string>
    </property>
   </widget>
   <widget class="QPushButton" name="outputConfigButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(52,73,85);
color:rgb(255,255,255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Configurar Salida</string>
    </property>
   </widget>
   <widget class="QPushButton" name="graphButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>791</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(52,73,85);
color:rgb(255,255,255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>GRAFICAR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputELine">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="filterELine">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputELine">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="GraphWidget" name="GraphWidget" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>691</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="nextGraphicButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>320</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(52,73,85);
color:rgb(255,255,255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prevGraphicButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>41</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(52,73,85);
color:rgb(255,255,255);
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphWidget</class>
   <extends>QWidget</extends>
   <header location="global">graphwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
